base/util.go

```diff
@@ -1120,3 +1120,31 @@ func injectJSONPropertyFromBytes(b []byte, bIsEmpty bool, kvPairs []KVPairBytes)
 
 	return newJSON
 }
+
+var UseStdlibJSON AtomicBool
```

Comment:
As discussed, can switch to a normal bool given we only have the single Set use case, and want to avoid overhead per-read.